aws-storage-s3/src/main/AndroidManifest.xml

```diff
@@ -21,5 +21,6 @@
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
     <application>
         <service android:name="com.amazonaws.mobileconnectors.s3.transferutility.TransferService" android:enabled="true" />
```

Comment:
@sdhuka I left this in place in case the escape hatch is being used, and the customer interacts with sdk directly. They may have previously relied on us importing the service, rather than adding to their own manifest, so it felt like a breaking change if we were to remove.